.github/workflows/pr-validate.yml

```
@@ -0,0 +1,16 @@
+name: "Validate PR"
```

At least for now, this is only validating the title of the PR - so would be good to be more specific about it I think.

```suggestion
name: "Validate PR title"
```

Might want to rename the workflow filename to reflect, too.